src/ParallelAlgorithms/Interpolation/Callbacks/ObserveTimeSeriesOnSurface.hpp

```diff
@@ -41,8 +41,24 @@ namespace callbacks {
 namespace detail {
 
 template <typename... Ts>
-auto make_legend(tmpl::list<Ts...> /* meta */) {
-  return std::vector<std::string>{"Time", db::tag_name<Ts>()...};
+auto make_legend(tmpl::list<Ts...>) {
```

Comment:
Please extend Test_ObserveTimeSeriesOnSurface.cpp so your new additions are tested.

Reply:
Did you mean Test_ObserveTimeSeriesAndSurfaceData.cpp? If so, should I add a test for all of the non-ylm tags; the tags found in /ApparentHorizon.dat in the .h5 file, because I only see tests for the ylm tags in /ApparentHorizon_Ylm.dat

Reply:
Hm no that file is testing something different. Unfortunately I don't find a good existing test of the other quantities, which seems like an oversight. Writing a whole new test is out of scope for this PR, so we'll just merge it like this.